habana/pytorch/buildspec.yml

```diff
@@ -27,30 +27,30 @@ context:
       target: deep_learning_container.py
 
 images:
-  BuildHabanaPytorchPy38SypAI130TrainingDockerImage:
+  BuildHabanaPytorchPy37SypAI120TrainingDockerImage:
```

Comment:
Should we make this as Py38 since the python version mentioned in the buildspec if Py38?